docs/csharp/language-reference/operators/null-conditional-operators.md

```diff
@@ -57,21 +59,18 @@ A?.B?.C?[0] == E
 var handler = this.PropertyChanged;  
 if (handler != null)  
     handler(…)  
-  
 ```  
   
 ```vb  
 Dim handler = AddressOf(Me.PropertyChanged)  
 If handler IsNot Nothing  
     Call handler(…)  
-  
 ```  
   
  The new way is much simpler:  
   
-```vb-c#  
+```csharp
 PropertyChanged?.Invoke(e)  
```

Comment:
Could you break this out into two separate code blocks (even though they're basically identical)?
```csharp
PropertyChanged?.Invoke(e);
```
```vb
PropertyChanged?.Invoke(e)
```